zyte_spider_templates/spiders/serp.py

```diff
@@ -58,6 +124,7 @@ class GoogleDomainParam(BaseModel):
 
 class GoogleSearchSpiderParams(
     MaxRequestsParam,
+    SerpItemTypeParam,
```

Comment:
Should we provide extract_from option, which applies only for the items extracted from the linked websites, not to search results extraction itself?

Reply:
Doesn't need to be in the same PR though.
We also should add custom attributes extraction, probably also separately.